src/lua/api-gateway/validation/oauth2/oauthTokenValidator.lua

```diff
@@ -166,15 +166,8 @@ function _M:getTokenFromCache(cacheLookupKey)
     return nil;
 end
 
--- imsAuth will validate the service token passed in "Authorization" header --
-function _M:validate_ims_token()
+function _M:validateOAuthToken(oauth_token, validation_config)
```

Comment:
I added think exact kind of initialization in the `validateRequest` method, which is calling this code, so this should cover your observation, right?
```
function _M:validateRequest(validation_config)
    validation_config = validation_config or {}
    validation_config.RESPONSES = validation_config.RESPONSES or RESPONSES;
```